gof/job.sh

```diff
@@ -11,10 +11,10 @@ echo "Channel:" $CHANNEL
 VARIABLE=$3
 echo "Variable:" $VARIABLE
 
-OUTPUT_DIR=/ceph/wunsch/jobs_gof_2016_2018-11-20/${ERA}_${CHANNEL}_${VARIABLE}
+OUTPUT_DIR=/ceph/mburkart/jobs_gof_2017_2020-01-31/${ERA}_${CHANNEL}_${VARIABLE}
```

Comment:
I would rather stick to this logic since it allows to have the results from previous iterations. One could consider to automatically generate the name of the directory from the user and the current date or add an option to give the output directory to the script.

Reply:
Changed to follow analysis in https://github.com/KIT-CMS/sm-htt-analysis/pull/45.